enterprise/server/cmd/goinit/main.go

```diff
@@ -452,6 +475,80 @@ func main() {
 	syscall.Reboot(syscall.LINUX_REBOOT_CMD_RESTART)
 }
 
+// Resizes the ext4 filesystem mounted at the given path to the given size.
+func resizeExt4FS(mountPath string, size int64) error {
+	s := &syscall.Statfs_t{}
+	if err := syscall.Statfs(mountPath, s); err != nil {
+		return status.InternalErrorf("statfs %s: %s", mountPath, err)
+	}
+	blocks := size / s.Bsize
```

Comment:
@maggie-lou to answer your question from the other PR, if the ext4 FS isn't perfectly aligned to the ext4 block size (4096) then this division will mean we have a few bytes of leftover data at the end of the file, which seems OK.

Reply:
Worst case it would be 4095B but agree that probably nothing would come up in practice